src/Equinor.ProCoSys.IPO.Infrastructure/Migrations/IPOContextModelSnapshot.cs

```diff
@@ -239,7 +239,8 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.Property<Guid>("Guid")
                         .HasColumnType("uniqueidentifier");
 
-                    b.Property<int>("InvitationId")
+                    b.Property<int?>("InvitationId")

```

Comment:
I don't understand why this got a .IsRequired line inserted here, do you?

Reply:
No, but if the database model is not changed, an int and a nullable required int should be the same.